bindings/java/tools/build.py

```diff
@@ -54,7 +55,11 @@ def get_cargo_artifact_name(classifier: str) -> str:
     parser.add_argument('--classifier', type=str, required=True)
     args = parser.parse_args()
 
-    cmd = ['cargo', 'build', '--color=always', '--release']
+    cmd = ['cargo', 'build', '--color=always']
+
+    # Only enable release build while OPENDAL_RELEASE is on.
+    if os.environ.get('OPENDAL_RELEASE') == 'on':
```

Comment:
Is this to speed up build on developing?

Reply:
Yep, build java binding requires 36m on windows at GHA.